Scripts/generate_test_plans.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+PROJECT_PATH="Bitwarden.xcodeproj"
+TEST_PLAN_UNIT_PATH="TestPlans/TestPlan-Unit.xctestplan"
+TEST_PLAN_SNAPSHOT_PATH="TestPlans/TestPlan-Snapshot.xctestplan"
+SNAPSHOT_TESTS_TARGET="BitwardenSharedTests"
+
+mint install atakankarsli/xctestplanner@v1.0.9
```

Comment:
I've decided not to include the installation of `xctestplanner` in the _Mintfile_ so it's only installed when actually being used. For now this script won't be called in CI/CD so there's no need to waste time installing `xctestplanner `.